Makefile

```diff
@@ -7,6 +7,7 @@ test-unit:
 .PHONY: test-unit
 
 test-e2e:
+	go test ./test/integration -run ^\(TestPullSecretExists\)$$ -timeout 2h
```

Comment:
do we need 2h here?

Reply:
I have created another PR with shorter period here: https://github.com/openshift/insights-operator/pull/135